gpu/gpu.py

```diff
@@ -0,0 +1,64 @@
+from random import random
+
+
+def check_squared_value(x, xSquer, testValue):
+    if x != testValue - 1:
+        return False
+    elif xSquer == 1:
+        return False
+    else:
+        return True
```

Comment:
Porównując to z alg na CPU to ten pierwszy warunek jest sprawdzany poza pętlą. Gdy x zostanie zmienione po wszystkich przejściach przez pętlę.
Ta zmiana psuje algorytm prawdopodobnie

Reply:
To może trzeba to po prostu wyrzucić poza zrównoleglenie?